src/App.jsx

```diff
@@ -89,7 +89,7 @@ function App(props) {
   const prevTaskLength = usePrevious(tasks.length);
 
   useEffect(() => {
-    if (tasks.length - prevTaskLength === -1) {
+    if (tasks.length < prevTaskLength) {
```

Comment:
In my version of `App.jsx`:

- The `<ul>` has a `role` of `list`.
- The `editTask` function has full comments; the one in this repo has comments incomplete or missing